AutoRest/TestServer/swagger/paging.json

```diff
@@ -18,7 +18,7 @@
   "paths": {
     "/paging/single": {
       "get": {
-        "x-ms-pageable": { "nextLinkName": "nextLink", "itemName": "values" },
+        "x-ms-pageable": { "nextLinkName": "next.Link", "itemName": "values" },
```

Comment:
This will cause a change in all the languages. Have you generated code for all the languages?